docs/ref/forms/api.txt

```diff
@@ -1448,7 +1477,10 @@ If you need to access some additional information about a form field in a
 template and using a subclass of :class:`~django.forms.Field` isn't
 sufficient, consider also customizing :class:`~django.forms.BoundField`.
 
-A custom form field can override ``get_bound_field()``:
+A custom form can override :attr:`~django.forms.Form.bound_field_class` and
+a custom form field can take an optional
+:attr:`~django.forms.Field.bound_field_class` constructor argument.
+Additionally, it can override ``get_bound_field()``:
```

Comment:
I am not sure that we should still recommend overriding `get_bound_field()`. Let's just point folk at the new way.